app/static/tests/unit/store/model/mutations.test.ts

```diff
@@ -1,15 +1,16 @@
 import { mutations } from "../../../../src/app/store/model/mutations";
 import { mockError, mockModelState } from "../../../mocks";
-import { RequiredModelAction } from "../../../../src/app/store/model/state";
 
 describe("Model mutations", () => {
     it("evaluates and sets odin runner", () => {
         const mockRunner = "() => 'runner'";
-        const state = mockModelState({ odinRunnerError: mockError("error") });
+        // TODO: need a test that shows we can clear this...
+        // const state = mockModelState({ odinRunnerError: mockError("error") });
+        const state = mockModelState();
 
         mutations.SetOdinRunner(state, mockRunner);
         expect((state.odinRunner as any)()).toBe("runner");
-        expect(state.odinRunnerError).toBe(null);
+        // expect(state.odinRunnerError).toBe(null); // TODO
```

Comment:
Are these still TODOs?

Reply:
this is the testing half of the above one, removed them now, and we'll come back to look at things if we get a situation with sticky errors that confuse people